Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -246,7 +246,7 @@ static void emit_includes_for_all_imports(auto& interface, auto& generator, bool
 
         generate_include_for(generator, interface->module_own_path);
 
-        if (is_iterator) {
+        if (is_iterator && !interface->fully_qualified_name.is_empty()) {
```

Comment:
This might just be a hack, but at the same time it seems fairly justified to verify that fully_qualified_name has a value before we try to generate an include statement.

Reply:
It's definitely a hack :)
Can we find out why there's ever an interface without a `fully_qualified_name` and fix that instead?

Reply:
But my argument is so convincing. Oh well, I'll look into it